cmd/sops/main.go

```diff
@@ -1583,6 +1583,10 @@ func main() {
 			Usage:  "do not check whether the current version is latest during --version",
 			EnvVar: "SOPS_DISABLE_VERSION_CHECK",
 		},
+		cli.BoolFlag{
+			Name:  "enable-version-check",
```

Comment:
I wonder if something like `check-for-updates` (or `has-updates`) would be a nicer way to phrase this.

Reply:
I changed it to `--check-for-updates`.